src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/JudicialUsersFunctionalTest.java

```diff
@@ -101,33 +98,26 @@ void shouldGet403WhenUserSearchApiToggledOff() {
         assertNotNull(errorResponse);
     }
 
-    @DisplayName("Scenario: Full list of Judicial user based on page size and page number")
+    @DisplayName("Scenario: Get Bad Request when ccdServiceName is ALL")
     @ParameterizedTest
     @ValueSource(strings = {"jrd-system-user", "jrd-admin"})
     @ExtendWith(FeatureToggleConditionExtension.class)
     @ToggleEnable(mapKey = REFRESH_USER, withFeature = true)
     void refreshUserProfile(String role) {
 
         RefreshRoleRequest refreshRoleRequest = RefreshRoleRequest.builder()
-                .ccdServiceNames("")
+                .ccdServiceNames("ALL")
                 .sidamIds(Collections.emptyList())
                 .objectIds(Collections.emptyList())
                 .build();
 
         Response refreshResponse = judicialApiClient.refreshUserProfiles(refreshRoleRequest, 1, 1,
                 "objectId", "ASC",role);
 
-        if (OK.value() == refreshResponse.getStatusCode()) {
-            List<UserProfileRefreshResponse> userProfiles = asList(refreshResponse.getBody()
-                    .as(UserProfileRefreshResponse[].class));
-            log.info("JRD get refreshResponse response: {}", userProfiles.get(0).getObjectId());
-            assertNotNull(userProfiles.get(0).getObjectId());
-        } else {
-            assertEquals(NOT_FOUND.value(),refreshResponse.getStatusCode());
-        }
+        assertEquals(BAD_REQUEST.value(),refreshResponse.getStatusCode());
```

Comment:
Looks like you have removed a success scenario and replaced with negative scenario. Now there are no success scenarios for refresh api. Can you add this as a separate test ?